clvm/operators.py

```diff
@@ -41,9 +41,18 @@ class OperatorDict(dict):
     operators can be added dynamically.
     """
 
-    def __new__(class_, d: Dict):
+    def __new__(class_, d: Dict, *args, **kwargs):
+        """
+        `quote_atom` and `apply_atom` must be set
+        `unknown_op_handler` has a default implementation
+        """
         self = super(OperatorDict, class_).__new__(class_, d)
-        self.unknown_op_handler = self.unknown_op_raise
+        self.quote_atom = kwargs["quote"] if "quote" in kwargs else d.quote_atom
+        self.apply_atom = kwargs["apply"] if "apply" in kwargs else d.apply_atom
+        if "unknown_op_handler" in kwargs:
+            self.unknown_op_handler = kwargs["unknown_op_handler"]
+        else:
+            self.unknown_op_handler = self.unknown_op_raise
```

Comment:
I think there should be a unit test for this constructor. ideally one that fails if I make a mistake rebasing my patch on top of this